test/jdk/sun/management/windows/exerevokeall.c

```diff
@@ -99,12 +99,13 @@ static char *getTextualSid(SID* sid) {
     }
 
     // S-SID_REVISION
-    sprintf(name, "S-%lu-", SID_REVISION );
+    snprintf(name, sizeof(name), "S-%lu-", SID_REVISION );
```

Comment:
Name is a `char *` so sizeof will return the size of the pointer, not the length of the allocate data.

Reply:
Ooooops.  Thank you for the catch!